### PR TITLE
Fix Backdrop issues

### PR DESCRIPTION
**Pull request by greven**
I've added BackdropTemplate to frames using the methods impacted by the API changes of 9.01.
Also bump version to current patch.

### CODE REVIEW

File: movable.lua [hunk 1]
```diff
@@ -463,8 +463,7 @@ do
 		if(not target:GetCenter()) then return end
 		if(backdropPool[target]) then return backdropPool[target] end
 
-		local backdrop = CreateFrame"Frame"
-		backdrop:SetParent(UIParent)
+		local backdrop = CreateFrame("Frame", nil, UIParent, BackdropTemplateMixin and "BackdropTemplate")
```
Comment (haste) [inline movable.lua:466]: No need to check that the backdrop mixin exists. Same change can be made to the others.
```suggestion
		local backdrop = CreateFrame("Frame", nil, UIParent, "BackdropTemplate")
```

**Review by haste — CHANGES_REQUESTED** [review]
Just needs some minor changes. Otherwise it's :cake:

File: movable.lua (hunk 1 above)
Comment (greven) [reply movable.lua:466]: I don't check for the existence of the Mixin in my own code, but isn't the goal of doing that making it retro-compatible?

**Comment by greven** [comment]
Changed the rest of the Mixin checks. :)
Should be fine now, I think, @haste.

Thanks for answering the PR so quickly btw! Appreciated.

**Review by haste — APPROVED** [review]

**Comment by haste** [comment]
Thanks for taking time to submit the fixes :blue_heart:

**Pull request merged**